### PR TITLE
Add Details about Testing the Project

### LINKED ISSUE #436: Add Information on how to test and build the project
Similar to what we have on https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md#setup

### PR DESCRIPTION
**Pull request by SudharakaP**
Fixes #436

Please make sure the below checklist is followed for Pull Requests.
  - [ ] Checks are green
  - [ ] Tests are added where necessary
  - [x] Documentation is added/updated where necessary
  - [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/jhipster-core/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -128,6 +128,18 @@ from the main (upstream) repository:
...
     git pull --ff upstream master
     ```
+## Development Setup
+
+First you have to fork the generator-jhipster project and [setup the development environment as detailed](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md#setup).
+Then fork the jhipster-core project and within it run `npm link` to create a symbolic link between the package
+folder and the global node modules folder. Afterwards navigate back to the cloned generator-jhipster folder and execute 
+`npm link jhipster-core`. 
+
+To summarize;
```
Comment (MathieuAA) [inline CONTRIBUTING.md]: ';' => ':'

Comment (MathieuAA) [reply CONTRIBUTING.md]: That's the only thing I could find, the rest is perfect

**Comment by MathieuAA** [comment]
Merging this, I'll commit directly on master to fix the typo

**Comment by SudharakaP** [comment]
> Merging this, I'll commit directly on master to fix the typo

Thanks :smile:

**Pull request merged**